utils/releaser/src/ReleaseWorker/Release/CreateAndPushGitTagsExceptProjectBaseReleaseWorker.php

```diff
@@ -0,0 +1,115 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Shopsys\Releaser\ReleaseWorker\Release;
+
+use PharIo\Version\Version;
+use Shopsys\Releaser\Packagist\PackageProvider;
+use Shopsys\Releaser\ReleaseWorker\AbstractShopsysReleaseWorker;
+use Shopsys\Releaser\Stage;
+
+final class CreateAndPushGitTagsExceptProjectBaseReleaseWorker extends AbstractShopsysReleaseWorker
+{
+    /**
+     * Packages that are not released - old packages or forks
+     * @var string[]
+     */
+    private const EXCLUDED_PACKAGES = [
+        // excluded from the initial tagging as there needs to be another commit with composer.lock and package-lock.json
+        // @see https://github.com/shopsys/shopsys/pull/1264
+        'shopsys/shopsys',
+        'shopsys/project-base',
+        // not maintained anymore
+        'shopsys/product-feed-interface',
+        'shopsys/phpstorm-inspect',
+        // forks
+        'shopsys/postgres-search-bundle',
+        'shopsys/doctrine-orm',
+        'shopsys/jparser',
+        // not related packages
+        'shopsys/syscart',
+        'shopsys/sysconfig',
+        'shopsys/sysreports',
+        'shopsys/sysstdlib',
+    ];
+
+    /**
+     * @var \Shopsys\Releaser\Packagist\PackageProvider
+     */
+    private $packageProvider;
+
+    /**
+     * @param \Shopsys\Releaser\Packagist\PackageProvider $packageProvider
+     */
+    public function __construct(PackageProvider $packageProvider)
+    {
+        $this->packageProvider = $packageProvider;
+    }
+
+    /**
+     * @param \PharIo\Version\Version $version
+     * @return string
+     */
+    public function getDescription(Version $version): string
+    {
+        return 'Create and push git tags for packages excluding monorepo and project-base';
+    }
+
+    /**
+     * Higher first
+     * @return int
+     */
+    public function getPriority(): int
+    {
+        return 640;
+    }
+
+    /**
+     * @param \PharIo\Version\Version $version
+     */
+    public function work(Version $version): void
+    {
+        $packages = $this->packageProvider->getPackagesByOrganization('shopsys', self::EXCLUDED_PACKAGES);
+        $packageNames = str_replace('shopsys/', '', $packages);
+
+        $versionString = $version->getVersionString();
+
+        $tempDirectory = trim($this->processRunner->run('mktemp -d -t shopsys-release-XXXX'));
+        $packageNamesWithProblems = [];
+        foreach ($packageNames as $packageName) {
+            $this->symfonyStyle->note(sprintf('Cloning shopsys/%s. This can take a while.', $packageName));
+            $this->processRunner->run(sprintf('cd %s && git clone https://github.com/shopsys/%s.git', $tempDirectory, $packageName));
+            $this->processRunner->run(sprintf('cd %s/%s && git checkout master && git tag %s', $tempDirectory, $packageName, $versionString));
```

Comment:
It would rather have the path as a single parameter:
```suggestion
            $this->processRunner->run(sprintf('cd %s && git checkout master && git tag %s', $tempDirectory . '/' . $packageName, $versionString));
```

Reply:
For me `%s/%s` is more readable because I know just from reading that I want to go into one particular folder in some other folder so I don't want to do this.